example-tags/tag-init.json

```diff
@@ -1,5 +1,6 @@
 {
     "Existing Tags": [ "Mathe", "Deutsch", "Englisch", "Latein", "Physik", "Biologie", "Geschichte", "Erdkunde"
```

Comment:
Could you rename "Existing Tags" to "Existing Subjects"? Because the "Exisiting Users" and "File Types" are also tags right?

Reply:
changed them